.env used during provisioning available to recipes.

MarkBennett opened the pull request:
This is useful for tools like foreman and runit which can use them when configuring the environment for tools on the box. Otherwise this env file needs to be reconstructed by recipes on the server.

This was the only other change in our plow file when I updated from your latest. Hopefully this is the last push related to setting up sentinel.

MarkBennett committed:
.env used during provisioning available to recipes.

This is useful for tools like foreman and runit which can use them when
configuring the environment for tools on the box. Otherwise this env
file needs to be reconstructed by recipes on the server.

mm53bar commented:
So the .env file is copied into .plow but is still added to the top of plow.sh?  Why not just `source` the copied file from plow.sh?

MarkBennett committed:
Source the env file from plow.sh.

Previously the contents of this file were being cat'ed into plow.sh
which seems unneccesary.

MarkBennett commented:
Updated to source env file instead of cat'ing it.

MarkBennett committed:
Comment on the purpose of the env file.
